src/chart/bar/BarView.js

```diff
@@ -135,20 +135,25 @@ export default echarts.extendChartView({
         var bgEls = [];
         var oldBgEls = this._backgroundEls || [];
 
+        var createBackground = dataIndex => {
```

Comment:
Arrow functions are not allowed in ECharts.

Reply:
Already changed to normal function in commit 2